src/components/AlertBox/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
```

Comment:
We don't need this because we already have an `Alert` in the SDS. Sorry, you spent time working on this!